index.html

```diff
@@ -26,6 +26,20 @@ <h1 id="logo">Aziz Mejri</h1>
         </ul>
       </nav>
     </header>
+     
+    <!-- Mobile pop-up menu -->
+ 
+    <div class="mb_menu">
+      <img src="media/png/x.pn" alt="x icon" id="x_icon">
+      <ul class="menu_list">
+        <li><a href="#projects" class="menu_list_items">Portfolio</a></li>
+        <li><a href="#about" class="menu_list_items">About</a></li>
```

Comment:
- [ ] Kindly consider updating the href attribute for this Anchor Link, currently, it points to a wrong Element that doesn't exist on the document.
```suggestion
        <li><a href="#about-me" class="menu_list_items">About</a></li>
```